dkms.in

```diff
@@ -1294,60 +1289,31 @@ do_install()
             local found_orginal=""
             for original_module in $archive_pref1 $archive_pref2 $archive_pref3 $archive_pref4; do
                 [[ -f $original_module ]] || continue
-                case "$running_distribution" in
-                    debian* | ubuntu* )
-                        ;;
-                    *)
-                        echo $"   - Found $original_module"
-                        echo $"   - Storing in $dkms_tree/$module/original_module/$kernelver/$arch/"
-                        echo $"   - Archiving for uninstallation purposes"
-                        mkdir -p "$dkms_tree/$module/original_module/$kernelver/$arch"
-                        mv -f "$original_module" "$dkms_tree/$module/original_module/$kernelver/$arch/"
-                        ;;
-                esac
+                echo $"Found pre-existing $original_module, archiving for uninstallation"
```

Comment:
This looks like it's changing the behavior on debian/ubuntu. Although if I'm reading the code correctly it didn't do a backup in these scenarios and this is actually an improvement.

Reply:
Yes, the backup is now executed on all distributions, not just Debian and Ubuntu.